src/routes/(app)/+page.svelte

```diff
@@ -54,11 +52,11 @@
 		origin_x = origin_xDefault;
 	}
 
-	function getSimulationLink(forCopy = false) {
-		let prefix = 'https://bbchalenge.org/';
-		if (!forCopy) {
-			prefix = '/';
```

Comment:
Notice: this feature is never actually used, and `bbchalenge` is misspelled. Hence it can be removed and we can just use a regular reactive value for this variable.